doc/developer-guides/hld/virtio-net.rst

```diff
@@ -484,7 +484,7 @@ optional):
 
 .. code-block:: none
 
-    -s 4,virtio-net,<tap_name>,[mac=<XX:XX:XX:XX:XX:XX>]
+    -s 4,virtio-net,<tap=name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Comment:
@weichenli-intel Need to mention mac_seed?

Reply:
Done